AsyncSwift/Views/MainTabView.swift

```diff
@@ -8,21 +8,26 @@
 import SwiftUI
 
 struct MainTabView: View {
+	@EnvironmentObject var appData: AppData
     var body: some View {
-        TabView {
+		TabView(selection: $appData.currentTab) {
             EventView()
                 .tabItem {
                     Label("Event", systemImage: "calendar")
                 }
+				.tag(Tab.event)
             
             TicketingView()
                 .tabItem {
                     Label("Ticketing", systemImage: "banknote")
                 }
+				.tag(Tab.ticketing)
+			
             StampView()
                 .tabItem {
                     Label("Stamp", systemImage: "checkmark.square")
                 }
+				.tag(Tab.stamp)
         }
     }
```

Comment:
```suggestion
        TabView {
            ForEach(Tab.allCases, id: \.self) { tab in
                tab.view.tabItem {
                    Image(systemName: tab.systemImageName)
                    Text(tab.title)
                }
            }
        }
```

```swift
enum Tab: String, CaseIterable {
     case event
     case ticketing
     case stamp

    var title: String {
        rawValue
    }

    var systemImageName: String {
        switch self {
        case .event: return "calendar"
        case .ticketing: return "banknote"
        case .stamp: return "checkmark.square"
        }
    }

    @ViewBuilder
    var view: some View {
        switch self {
        case .event: EventView()
        case .ticketing: TicketingView()
        case .stamp: StampView()
        }
    }
 }
```

이렇게 enum으로 Tab을 관리하면 어떨지 제안드려봅니다 ~